src/Raven.Client/Http/DatabaseTopologyLocalCache.cs

```diff
@@ -104,6 +104,18 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, stream))
                         WriteNode(writer, node, context);
                     }
                     writer.WriteEndArray();
+                    
+                    writer.WriteComma();
+                    writer.WritePropertyName(context.GetLazyString(nameof(topology.Promotables)));
+                    writer.WriteStartArray();
+                    for (var i = 0; i < topology.Promotables.Count; i++)
+                    {
+                        var node = topology.Promotables[i];
+                        if (i != 0)
+                            writer.WriteComma();
+                        WriteNode(writer, node, context);
+                    }
+                    writer.WriteEndArray();
```

Comment:
I couldn't find any tests that checked the topology cache. The related ones I could find passed even when I purposely corrupted this json. So nothing is checking this now..